autogpts/autogpt/autogpt/command_decorator.py

```diff
@@ -19,52 +13,58 @@
 
 
 def command(
```

Comment:
The command decorator was reworked and simplified. Now components provide commands and it's up to them to check if any particular command should be passed to the agent or not. `name` and `aliases` are just `names` now and if not provided a function name will be used. Similarly if `description` isn't provided, first part of docstring is used.
Now only parameters are compulsory but this could also be reworked so the docstring `Args:` are used.
`is_valid` is problematic and could be also removed.